client/src/hooks/useAutocomplete.js

```diff
@@ -115,6 +119,44 @@ const useAutocomplete = ({
     [immediateResponses, setImmediateResponseAutocomplete]
   );
 
+  const checkInvolvesChild = useCallback(
+    (desc) => {
+      if (involvesChildOptions) {
+        const involvesChild = autocompleteSingleOption(
+          desc,
+          involvesChildOptions
+        );
+        if (involvesChild) {
+          setInvolvesChildAutocomplete(involvesChild);
```

Comment:
Not sure what values we're storing in Sanity for the autocomplete here, but I feel like this is inconsistent with line 133.  Is there any reason as to why we can pass in the value directly here, but we have to filter on line 133?